ui/Rhino/AGS/dev/AGS_load_session_cmd.py

```diff
@@ -0,0 +1,71 @@
+from __future__ import print_function
+from __future__ import absolute_import
+from __future__ import division
+
+import os
+import json
+
+import scriptcontext as sc
+
+import compas_rhino
+from compas.utilities import DataDecoder
+
+from compas_ags.diagrams import FormDiagram
+from compas_ags.diagrams import ForceDiagram
+
+
+__commandname__ = "AGS_load_session"
```

Comment:
i know i suggested this name, but one could argue that it makes more sense in terms of logical grouping of commands that it should be "AGS_session_load"